src/main/java/it/gov/pagopa/receipt/pdf/notifier/ReceiptToIO.java

```diff
@@ -98,33 +100,40 @@ public void processReceiptToIO(
         List<IOMessage> messagesNotified = new ArrayList<>();
 
         listReceipts.parallelStream().forEach(receipt -> {
-            if (isReceiptNotValid(receipt)) {
-                return;
+            try {
```

Comment:
For readability, could the `parallelStream` logic be included in an ad-hoc method?